lib/TimeZoneUtil.php

```diff
@@ -16,6 +16,10 @@ class TimeZoneUtil
 {
     public static $map = null;
 
+    public static $microsoftMap = [
+        'tzone://Microsoft/Utc' => 'UTC',
```

Comment:
Can you add this entry in the extrazones file instead? It's important to keep all the conversions in the same place